dotcom-rendering/src/web/components/SubNav.importable.tsx

```diff
@@ -155,10 +155,11 @@ export const SubNav = ({ subNavSections, currentNavLink, format }: Props) => {
 
 	useEffect(() => {
 		const ulEl = ulRef.current;
-		if (ulEl) {
-			const lis = ulEl.querySelectorAll('li');
-			const lastLi = lis[lis.length - 1];
 
+		const lis = ulEl?.querySelectorAll('li');
+		const lastLi = lis && lis[lis.length - 1];
```

Comment:
```suggestion
		const lastLi = lis?.at(-1);
```

Reply:
Not supported on our Node version, doesn't get transpiled :(

Reply:
Even the `slice` approach?

Reply:
Slice approach did work, posted that comment before you edited :)

Reply:
Ah, yeah, makes sense! Thanks for humoring me!